.platform.yml

```diff
@@ -1 +1,3 @@
 infrastructure: ^3
```

Comment:
The `infrastructure` key isn't in [the docs](https://servicedesk.silverstripe.cloud/support/solutions/articles/75000012884) so I can't tell if this does anything, and if it does, if it will in any way conflict with the php version defined below.

My best guess is that it's a legacy key for tracking something about the 3rd iteration of platform (see https://servicedesk.silverstripe.cloud/support/solutions/articles/75000012882-infrastructure-changelog we're on 4 right now) so it should probably be removed?